### ISSUE TITLE
b-form-radio-group (and others?) now set the model value to "" instead of leaving it null/undefined

### ISSUE THREAD

**Issue by mariusa** [bootstrap-vue/bootstrap-vue-next#1003]
**Describe the bug.** A clear and concise description of what the bug is.

Until version 0.8, one could do this:
```

let conditionType = $ref()
console.log("conditionType initial", conditionType, typeof conditionType)
// later in code
setTimeout(() => {
    console.log("conditionType later", conditionType, typeof conditionType)
    conditionType ??= 'a'
    console.log("conditionType later2", conditionType)
}, "1000")



// template
<b-form-radio-group
                            v-model="conditionType"
                            :options="[{value: 'a', text: 'AA'}, {value: 'b', text: 'BB'}]"
                        />

```

Result
```
conditionType initial undefined undefined
conditionType later  string
conditionType later2 
```

b-form-radio-group (and others?) now set the model value to "" instead of leaving it null/undefined
In an older version, the code above worked, `conditionType later ` being null or undefined. After that code, `conditionType` becomes  `'a'` and appears selected by default.

**Expected behavior.** A clear and concise description of what you expected to happen.
b-form-radio-group (and others?) shouldn't set a "" value for a model where user didn't select anything yet. "" is an invalid value when an array of options is given. Instead, leave it null or undefined until either it's set programatically or user selects something.

**Screenshots.** If applicable, add screenshots to help explain your problem.

- OS: Linux
- Package Version: 0.8.0
- Npm/Yarn/Pnpm Version: npm 9

**Comment by VividLemon** [comment]
I'm not sure how that code would have ever worked before, the default for modelValue has never changed.

**Comment by xvaara** [comment]
Yes, modelValue default value should be undefined (or null?) in My opinion. Or modelValue should be required without default? Is there reason to have a form elements without v-model?

it was changed when doing conversion to script setup: https://github.com/bootstrap-vue/bootstrap-vue-next/commit/9726d1004e0da2f04b8222961b7c234790ea0125

**Comment by VividLemon** [comment]
How do we feel that checkbox groups should behave? How does Bootstrap-vue implement this? Do they set their default value as null? Or is it an empty array?

**Comment by mariusa** [comment]
IMO, checkbox groups, a list of potential values, should be an empty array by default.
